bigquery_etl/format_sql/format.py

```diff
@@ -141,6 +141,7 @@
     "sql/moz-fx-data-shared-prod/udf_js/jackknife_mean_ci.sql",
     "sql/moz-fx-data-shared-prod/udf_js/jackknife_ratio_ci.sql",
     "sql/moz-fx-data-shared-prod/udf_js/jackknife_sum_ci.sql",
+    "sql/moz-fx-data-shared-prod/udf_js/jackknife_percentile_ci/udf.sql",
```

Comment:
The final `udf.js` is necessary here. We changed the directory structure for UDFs at some point, so the files above refer to old files that have been moved. I removed the references here to avoid any future confusion.
Also, skipping formatting for the new UDF here doesn't seem to be necessary.